inmem_transport_test.go

```diff
@@ -1,7 +1,9 @@
 package raft
 
 import (
+	"github.com/stretchr/testify/require"
```

Comment:
move me

Reply:
This is an upstream commit. Leaving unchanged.